packages/front-end/components/Features/RequestReviewModal.tsx

```diff
@@ -163,6 +164,36 @@ export default function RequestReviewModal({
   } else if (canReview) {
     ctaCopy = "Next";
   }
+  const showRevisionStatus = () => {
```

Comment:
these should probably be larger / bolder compared to the detailed log view below